src/stream/easy.rs

```diff
@@ -203,7 +217,10 @@ pub enum Error<T, R> {
     /// Generic message
     Message(Info<T, R>),
     /// Variant for containing other types of errors
+    #[cfg(feature = "std")]
     Other(Box<dyn StdError + Send + Sync>),
+    #[cfg(all(feature = "alloc", not(feature = "std")))]
+    Other(Box<dyn DebugAndDisplay + Send + Sync>),
```

Comment:
Features should be strictly additive so compiling this to variants with different types dependent on feature flags is not something I want to do. Waiting for the error trait to get moved into core would be best though I do not know of any timeline on that.

Reply:
I see solution as a custom error type (something like anyhow::Error) for this variant.
But it's a breaking change

Reply:
If you revert this part (effectively keeping "easy errors" on `std` still I 'd be happy to merge the rest.